### PR TITLE
Rita alley messages 02

### PR DESCRIPTION
**Pull request by ritamwaikinda**

### CODE REVIEW

File: server/db/models/userModel.js [hunk 1]
```diff
@@ -50,22 +48,7 @@ const UserSchema = new mongoose.Schema(
...
-    type: Number,
-    required: true,
-    trim: true,
-    validate(value) {
-      if (!value) {
-        throw new Error('Zip code is empty');
-
-      }
-    ,
+    ],
```
Comment (saritasalazar) [inline server/db/models/userModel.js:51]: Hey Ianne! I think you might have an a random square closing bracket on line 51.

**Review by saritasalazar — CHANGES_REQUESTED** [review]
I think I spy a random square bracket in your code. I left a comment on the line. Take a look and remove if needed.

File: server/db/models/userModel.js (hunk 1 above)
Comment (ritamwaikinda) [reply server/db/models/userModel.js:51]: Hi Sara. Ah, I believe we are looking to make "location" an array of co-ordinates (starting at line 44).
But if it's still don't wrong, please let us know

Comment (saritasalazar) [reply server/db/models/userModel.js:50]: Perfect - thank you! Just wanted to double check. I couldn't find the opening bracket for some reason.

**Review by saritasalazar — APPROVED** [review]

File: server/db/models/userModel.js (hunk 1 above)
Comment (ritamwaikinda) [reply server/db/models/userModel.js:50]: I feel you, it happened to me too! I believe Github was trying to be helpful, only showing things that had changed and hiding things that hadn't. but nooope, not helpful!
Thank you, Sara!!!

**Pull request merged**